tfe/resource_tfe_workspace.go

```diff
@@ -76,7 +76,7 @@ func resourceTFEWorkspace() *schema.Resource {
 			"working_directory": {
 				Type:     schema.TypeString,
 				Optional: true,
-				Computed: true,
+				Default:  "",
```

Comment:
Since we now always expect (either the default or a supplied) value here, we also need to always set the value. So this means that we need to ad this option to this block: https://github.com/terraform-providers/terraform-provider-tfe/blob/0177cd44722d1077197731b5f4fd67d9230fcf71/tfe/resource_tfe_workspace.go#L129-L135

And then remove this block: https://github.com/terraform-providers/terraform-provider-tfe/blob/0177cd44722d1077197731b5f4fd67d9230fcf71/tfe/resource_tfe_workspace.go#L148-L150

Reply:
Good catch. I will make this change.